server/api.go

```diff
@@ -70,6 +74,8 @@ func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Req
 		p.completeConnectUserToGitlab(w, r)
 	case "/api/v1/connected":
 		p.getConnected(w, r)
+	case "/api/v1/getsettings":
```

Comment:
```suggestion
	case "/api/v1/settings":
```

Reply:
@mickmister already a route is present with same https://github.com/mattermost/mattermost-plugin-gitlab/blob/64570f1377e7f9712d08536e484ee7d09293bdf5/server/api.go#L89

Reply:
How about `client_settings` instead?